### DISCUSSION TITLE
Datenrettung trotz Fatal error

### DISCUSSION THREAD

**Discussion by mctheis** [PapooSoftware/PapooCMS#15] [Q&A] [answered]
Hallöchen,
aus dem Nichts geht meine Seite und leider auch die eines Bündnisses nicht mehr. Meine Seite ist unwichtig aber ich brauche die Inhalte der Bündnisseite!!! Viele Jahre hat mir papoo immer geholfen, wenn irgendetwas war aber die wollen ja nicht mehr (seufz). Wenn ich wenigstens in den Interna-Bereich hereinkommen würde, könnte ich die Texte herauskopieren. Leider geht nichts mehr.
Habe jetzt ein Verzeichnis im Verzeichnis angelegt, um zu testen, ob ich irgendwie an die Inhalte herankomme....leider habe ich weniger Fehlermeldungen aber drin bin ich dennoch nicht.

URL http://bilk-wohnen-fuer-alle.de/neu_bilk-wohnen-fuer-alle.de
Fatal error: Array and string offset access syntax with curly braces is no longer supported in /www/htdocs/w0192608/bilk-wohnen-fuer-alle.de/lib/function.php on line 306

Warum betrifft der Fatal Error nicht die neu_bilk-wo.... ??? Ich habe ebenso die function.php nach geschweiften Klammern durchsucht, mehrmals die neueste Datei aus papoo23.04 drüber gespielt und...ach, Hilfe bitte, bin nur eine kleine Designerin und habe in Informatik nur gefährliches Halbwissen.

Papoo hat mir geraten, ein anderes CMS zu nutzen.... schade, war großer Fan, jedenfalls brauche ich jetzt die Inhalte, um ggf. irgendein CMS zu finden, wo ich die einsetzen kann.

Danke schonmal und VG

**Comment by kapudo** [comment]
Moin...

- snip -
"aber die wollen ja nicht mehr (seufz)"
"Papoo hat mir geraten, ein anderes CMS zu nutzen"
- snip -

Die Aussagen verstehe ich gerade nicht. Das CMS wird doch gepflegt (?).... 

VG Markus

**Comment by ah-papoo** [comment, marked as answer]
Hallo mctheis,

> Viele Jahre hat mir papoo immer geholfen, wenn irgendetwas war aber die wollen ja nicht mehr (seufz
Diese Aussage ist gänzlich falsch, ich hatte Ihnen per Mail mitgeteilt, dass die Fehlermeldung darauf schließt, dass Sie das Update nicht korrekt durchgeführt haben und u.A. diese Datei nicht ersetzt wurde aus der aktuellen Version, wenn Sie mir dann nicht mehr per Mail antworten, dass die Ersetzung dieser Datei nicht geholfen hat, ist es mir gänzlich unmöglich Ihnen zu helfen. Ich würde vorschlagen, dass Sie mich einfach mal unter unserer regulären Nummer anrufen und ich versuche Ihnen bei dem Problem zu helfen.

> Papoo hat mir geraten, ein anderes CMS zu nutzen
Meine Aussage war: "Da wir an dem Papoo-CMS nicht mehr entwickeln und Kunden bereits sagen, dass Sie sich ein anderes CMS suchen wollen, würde sich ein Update der Dokumentation hier nicht mehr lohnen."
Dies hat nichts damit zu tuen, ob wir eventuelle Fehler im CMS behoben oder den Kunden die das CMS nutzen noch helfen und zieht einfach die falschen Schlüsse. Wir teilen bereits aktiv unseren Kunden mit, dass wir vermutlich in der Zukunft hier am CMS keinen großen Updates mehr machen werden (Neue PHP-Versionen) und nur noch Anpassungen einfügen, welche wir ohnehin von anderen Kunden für beauftragt wurden.

Grüße
Achim

**Reply by mctheis** [reply]
DANKE!!! Alles gerettet! Ich bin soooo erleichtert!
